### ISSUE TITLE
Use Flutter RSA with NodeJs

### ISSUE THREAD

**Issue by rupamking1** [AbdelbakiBoukerche/rsa_encrypt#3]
Use Flutter RSA with NodeJs
I want to encrypt message in Flutter and Decrypt in Nodejs, I already tryed 5-7 Nodejs Packages but I got error.
Please help me.

**Comment by nullptr-31** [comment]
Hello, Thanks for showing interest in rsa_encrypt.
For now there's no way u can decrypt strings encrypted with rsa_encrypt
outside flutter application. I made rsa_encrypt for a personal project
where I needed to encrypt data send it to server get the data(encrypted)
and decrypt it before showing to the use. This is how rsa_encrypt and this
is what its made to do thus its impossible to decrypt any data outside the
flutter app.
This is the 2nd issue regarding this problem and unfortunately I stopped
rsa_encrypt development. But I might add a way to encrypt data with it and
decrypt it outside the app.

Happy coding.
